shared-test/src/main/java/uk/gov/di/authentication/sharedtest/helper/AuditAssertionsHelper.java

```diff
@@ -27,6 +27,45 @@ public static void assertNoTxmaAuditEventsReceived(SqsQueueExtension txmaAuditQu
                         () -> assertThat(txmaAuditQueue.getApproximateMessageCount(), equalTo(0)));
     }
 
+    public static void assertUnvalidatedTxmaAuditEventsReceived(
```

Comment:
how about something like `assertTxmaAuditEventsSubmittedWithMatchingNames`?
I guess reading the `unvalidated` bit of this in a test would be a bit surprising to me, I think I'd spend a bit of time working out what was meant by unvalidated